docs/optimization_guide/nncf/ptq/basic_quantization_flow.md

```diff
@@ -29,78 +29,118 @@ Install all the packages required to instantiate the model object, for example,
+         :fragment: [dataset]
 
-.. tab:: ONNX
+   .. tab-item::  PyTorch
+      :sync: pytorch
+      
+      .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_torch.py
+         :language: python
+         :fragment: [dataset]
 
-    .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_onnx.py
-       :language: python
-       :fragment: [dataset]
+   .. tab-item:: ONNX
+      :sync: onnx
 
-.. tab:: OpenVINO
+      .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_onnx.py
+         :language: python
+         :fragment: [dataset]
 
-    .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_openvino.py
-       :language: python
-       :fragment: [dataset]
+   .. tab-item:: TensorFlow
+      :sync: tensorflow
 
-.. tab:: TensorFlow
+      .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_tensorflow.py
+         :language: python
+         :fragment: [dataset]
 
-    .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_tensorflow.py
-       :language: python
-       :fragment: [dataset]
+If there is no framework dataset object, you can create your own entity that implements the ``Iterable`` interface in Python, for example the list of images, and returns data samples feasible for inference. In this case, a transformation function is not required.
 
 
-If there is no framework dataset object, you can create your own entity that implements the ``Iterable`` interface in Python and returns data samples feasible for inference. In this case, a transformation function is not required.
-
-
-Run a Quantized Model
+Quantize a Model
 #####################
 
 Once the dataset is ready and the model object is instantiated, you can apply 8-bit quantization to it:
 
-.. tab:: PyTorch
+.. tab-set::
+
+   .. tab-item:: OpenVINO
+      :sync: ov
+      
+      .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_openvino.py
+         :language: python
+         :fragment: [quantization]
+
+   .. tab-item::  PyTorch
+      :sync: pytorch
+      
+      .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_torch.py
+         :language: python
+         :fragment: [quantization]
+
+   .. tab-item:: ONNX
+      :sync: onnx
+
+      .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_onnx.py
+         :language: python
+         :fragment: [quantization]
 
-    .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_torch.py
-       :language: python
-       :fragment: [quantization]
+   .. tab-item:: TensorFlow
+      :sync: tensorflow
 
-.. tab:: ONNX
+      .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_tensorflow.py
+         :language: python
+         :fragment: [quantization]
+         
 
-    .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_onnx.py
-       :language: python
-       :fragment: [quantization]
+After that the model can be converted into the OpenVINO Intermediate Representation (IR) if needed, compiled and run with OpenVINO:
```

Comment:
> It would be good to also explain how to save the quantized model to disk. Like `ov.serialize(quantized_model, "quantized_model.xml")`

It looks like this is common OpenVINO API.

Reply:
It is! Just like the OpenVINO inference code above. Not every person who learns how to quantize a model with NNCF will be familiar with how to serialize models in OpenVINO, so it's good to document it.